src/corehost/common/pal.h

```diff
@@ -223,6 +226,8 @@ namespace pal
     bool load_library(const string_t* path, dll_t* dll);
     proc_t get_symbol(dll_t library, const char* name);
     void unload_library(dll_t library);
+
+    bool is_running_in_wow64();
```

Comment:
I think this belongs under `#if WIN32`

Reply:
I elected to have the declaration available globally